README.md

```diff
@@ -19,6 +19,11 @@ If you are already using a linter to identify coding standards problems in your
 code, you know that fixing them by hand is tedious, especially on large
 projects. This tool does not only detect them, but also fixes them for you.
 
+## Supported versions
+* PHP 7.4
+* PHP 8.0
+* PHP 8.1
```

Comment:
I wonder if we can put some guard (eg in `tests/AutoReview/`) to make sure that this list is up to date ?

Reply:
Hm, it would be cool, I'll try to look at it

Reply:
I think 7910bf5 should be enough to notice any change in supported PHP versions